ci/spelling.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+npm install -g cspell
+git fetch origin master:master
+git diff --name-only master $HEAD | xargs -L1 npx cspell -c ./ci/cspell.json -u -e ci/
```

Comment:
We don't need to execute `npm install`  when we use `npx` as the latter installs dependencies automatically. Quote from https://www.npmjs.com/package/npx

> Executes <command> either from a local node_modules/.bin, or from a central cache, installing any packages needed in order for <command> to run

Reply:
Hmm agreed that this script may be more explicit than it needs to be. Thanks for the review!